src/sdk/namespace_fs.js

```diff
@@ -1006,11 +1006,13 @@ class NamespaceFS {
         const { source_stream } = params;
         try {
             // Not using async iterators with ReadableStreams due to unsettled promises issues on abort/destroy
+            const MD5Enabled = params.force_md5_etag
```

Comment:
just to be consistent with our coding style 
```suggestion
            const md5_enabled = params.force_md5_etag
```

Reply:
done, Thanks